src/cli/tc.rs

```diff
@@ -37,9 +41,55 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init { file_path } => Ok(init(&file_path).await?),
   }
 }
 
+pub async fn init(file_path: &str) -> Result<()> {
+  let tailcallrc = resource_str!("assets/.tailcallrc.graphql");
+
+  let ans = Confirm::new("Do you want to add a file to the project?")
+    .with_default(false)
+    .prompt();
+
+  match ans {
+    Ok(true) => {
+      println!("{}", "Enter the file name:".yellow());
+      let mut file_name = String::new();
+      io::stdin().read_line(&mut file_name)?;
+      file_name = format!("{}.graphql", file_name.trim());
+
+      let confirm = Confirm::new(&format!("Do you want to create the file {}?", file_name))
+        .with_default(false)
+        .prompt();
+
+      match confirm {
+        Ok(true) => {
+          fs::write(format!("{}/{}", file_path, &file_name), "")?;
+
+          let graphqlrc = format!(
+            r#"schema:
+- "./{}"
+- "./.tailcallrc.graphql"#,
```

Comment:
```suggestion
            r#"|schema:
               |- "./{}"
               |- "./.tailcallrc.graphql"#.strip_margin(),
```

Reply:
Tried `stripmargin` crate with `strip_margin()`,  `strip_margin_with()` didn't work for some reason when I ran it, `|` wasn't stripped.
Tried quite a few variations.

Reply:
@tusharmath Fixed it!, wasn't aware single quotes was valid YAML markup coming from JSON lol. Sorry about that.